### PR TITLE
fix: fix the issue with the progress bar not updating properly

### PR DESCRIPTION
**Pull request by InzamZ**
The `pbar.update(delta)` method updates not the index but the increment. As the total count of pbar is a label `<p>`, the position of `pbar.update(delta)` is also incorrect. Therefore, it happens to display correctly in the example.

`pbar.update(delta)` 方法更新的不是下标而是增量。`pbar` 的总数量是标签 `<p>`，因此 `pbar.update(delta)` 的位置也是错误的，所以在示例中恰好显示正常

### CODE REVIEW

**Comment by InzamZ** [comment]
Fixing #79

File: make_book.py [hunk 1]
```diff
@@ -174,9 +174,11 @@ def make_bilingual_book(self):
         p_to_save_len = len(self.p_to_save)
         try:
             for i in self.origin_book.get_items():
-                pbar.update(index)
                 if i.get_type() == ITEM_DOCUMENT:
-                    soup = bs(i.content, "html.parser")
+                    if i.file_name.endswith(".xhtml"):
+                        soup = bs(i.content, "xml")
+                    else:
+                        soup = bs(i.content, "html.parser")
```
Comment (yihong0618) [inline make_book.py:181]: use this logic?

Comment (InzamZ) [reply make_book.py:181]: 我重构一下

Comment (InzamZ) [reply make_book.py]: https://github.com/yihong0618/bilingual_book_maker/blob/4b21d6f523270193d14561efb764a0f27d50864c/make_book.py#L166-L171

这部分逻辑是否有错误

Comment (InzamZ) [reply make_book.py]: 还是我理解有问题🥲

File: make_book.py [hunk 2]
```diff
@@ -174,9 +174,12 @@ def make_bilingual_book(self):
...
         try:
             for i in self.origin_book.get_items():
-                pbar.update(index)
                 if i.get_type() == ITEM_DOCUMENT:
-                    soup = bs(i.content, "html.parser")
+                    soup = (
+                        bs(i.content, "xml")
+                        if i.file_name.endswith(".xhtml")
+                        else bs(i.content, "html.parser")
+                    )
```
Comment (yihong0618) [inline make_book.py:182]: cool

**Pull request merged**